### DISCUSSION TITLE
Welcome to Minecraft-SMP-Bot Discussions!

### DISCUSSION THREAD

**Discussion by Lianecx** [MC-Linker/MC-Linker#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

If you need futher help, make sure to join our support server: https://discord.gg/rX36kZUGNK

**Comment by Jojoshua903** [comment]
how to enable /inventory?

**Reply by Lianecx** [reply]
You can manage all commands in `Server Settings > Integrations > MC Linker`. If you've disabled it using `/disable` you can just enable it back using `/enable`. If you need more help, please join our support server: https://discord.gg/rX36kZUGNK
